variables.tf

```diff
@@ -21,4 +21,43 @@ variable "at_create_instance" {
   default     = true
 }
 
-********************************************************************/
\ No newline at end of file
+********************************************************************/
+
+
+variable "enable" {
+  default     = true
```

Comment:
Prefix bool variables with   "is_" ,  which is like asking user YES/NO ?

Reply:
added is_

Reply:
Can we rename this variable to something like provison_cp4s , to reflect the functionality it supports